pkg/apis/sriovnetwork/v1/sriovnetwork_types.go

```diff
@@ -36,6 +36,12 @@ type SriovNetworkSpec struct {
 	// VF link state (enable|disable|auto)
 	// +kubebuilder:validation:Enum={"auto","enable","disable"}
 	LinkState string `json:"linkState,omitempty"`
+	// +kubebuilder:validation:Minimum=0
+	// Minimum tx rate, in Mbps, for the VF. Defaults to 0 (no rate limiting). min_tx_rate should be <= max_tx_rate.
+	MinTxRate *int `json:"min_tx_rate"`
```

Comment:
Please consider using `minTxRate` instead of `min_tx_rate` to follow the naming convention of CRD. And also for the `maxTxRate`.

Reply:
Done. Thanks.